### ISSUE TITLE
Is collapsePhrase available for displacy?

### ISSUE THREAD

**Issue by pengyu** [explosion/spaCy#2266]
I don't see `collapsePhrase` available for displacy. Although it is available on https://github.com/explosion/displacy

Could anybody confirm whether `collapsePhrase` is available in displacy? If not, can it be added to make it consistent with displacy.js?

**Comment by ines** [comment]
TL;DR: No, not yet. I actually noticed this too, while re-doing the demos (but I didn't yet have time to add it, and I forgot to open an issue myself).

You can find the source of the built-in `displacy` visualizer [in `spacy/displacy/__init__.py`](https://github.com/explosion/spaCy/blob/master/spacy/displacy/__init__.py#L80).

The function used to collapse phrases is pretty simple – it's currently only implemented on the back-end [as part of the displaCy service](https://github.com/explosion/spacy-services/blob/3bb2caa23110a76d7f403287d76b42d19bcc04f0/displacy/app.py#L39-L42):

```python
if collapse_phrases:
    for np in list(doc.noun_chunks):
        np.merge(tag=np.root.tag_, lemma=np.root.lemma_,
                 ent_type=np.root.ent_type_)
```

All it's doing here is iterating over the noun chunks and merging them, with the phrase root's tag, lemma and entity type set.
